index.html

```diff
@@ -1,3 +1,39 @@
----
-layout: resume
----
\ No newline at end of file
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8"> <title>TODO list</title> <style>
+h1 { text-align:center;
+}
+h3 {
+text-transform: uppercase; }
+ul {
+                margin: 0;
+padding: 0; }
+            ul li {
+                cursor: pointer;
+position: relative;
+padding: 12px 8px 12px 40px; background: #eee;
+font-size: 18px;
+transition: 0.2s;
+}
+ul li:nth-child(odd) {
+                background: #f9f9f9;
+            }
+            ul li:hover {
```

Comment:
```suggestion
            ul li:hover { maby
```
use better `user_sekeet: none`

Reply:
ok